kahuna/public/js/search/query.js

```diff
@@ -82,6 +82,9 @@ query.controller('SearchQueryCtrl',
     }
 
     $scope.$watchCollection(() => ctrl.filter, onValChange(filter => {
+        if (filter.query.indexOf('~') === 0) {
```

Comment:
This is quite simplistic (if `~` is in a word or a quoted phrase, it will incorrectly detect it as a collection filter). We should at least add a comment to that effect. Or maybe we should only detect it as a collection if the query starts with `~`?